tasks/pre.yml

```diff
@@ -66,17 +66,67 @@
   tags:
     - fact
 
-- name: redhat family epel installation
+- name: RedHat family (Centos etc..) EPEL installation
   become: 'yes'
   dnf:
     name: "epel-release"
     state: present
     update_cache: 'yes'
-  when: ansible_os_family == "RedHat" and (not ansible_distribution == "Fedora")
+  when: ansible_os_family == "RedHat" and (not (ansible_distribution == "Fedora" or ansible_distribution == "RedHat"))
   tags:
     - dnf
     - packages
 
+- name: RHEL8 repo tasks
+  block:
+    - name: Install RHEL8 EPEL GPG key
+      become: 'yes'
+      ansible.builtin.rpm_key:
```

Comment:
not started with the .10 conversions yet. still needs to be .9 supported.

Reply:
Hallo. Maybe I don't catch what do you mean exactly by the comment. But the role do not support RHEL yet, only CentOS, isn't it? So the question is if you want to extend the role by this way - to provide tests and other tasks you have to configure an real RHEL system etc..  I made decision to use RHEL for my homelab and other systems.  I'm leaving CentOS, because 16 licenses for RHEL is more then enough for me. It's a reason of my last modifications... But maybe it not fit your needs and then my PR can be contra productive. Please let me know...

Reply:
No, I meant `ansible.builtin.rpm_key` is the "format" for Ansible 2.10

Reply:
My ansible version is ansible 2.9.15.  Seems that it is able to handle this syntax without problems....

Reply:
Yeah it does, but my plan is to convert to FQCN when the 2.11 beta arrives so I'm sure to support Latest stable - 1.
This way there's no need to mix and match at the moment.

https://docs.ansible.com/ansible/devel/roadmap/ROADMAP_2_11.html